static/js/around-map.js

```diff
@@ -173,7 +173,14 @@ function scrollCard (poi_id) {
   window.scrollTo(0, (cardTop - cardHeight/2))  
 }
 
-// saving/adding a spot - form
+// pois form
+const displayForm = document.querySelectorAll('.display-form')
+displayForm.forEach( button => {
```

Comment:
because the button to open or to close have a class name `.display-form`
So I use it to toggle a class `visible`on the form
But I'll change